me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeWebVerify.java

```diff
@@ -78,64 +74,69 @@ public void startWebView() {
     }
 
     if (start) {
-      String url = createURL();
+      loginGetAccessTokenListeners = listener;
+      String url = createURL(activity, scope);
       intent.putExtra(WebViewActivity.EXTRA_URL, url);
       intent.putExtra(WebViewActivity.EXTRA_SCOPE, scope);
-      intent.putExtra(WebViewActivity.EXTRA_CLIENT_ID, clientID);
-      intent.putExtra(WebViewActivity.EXTRA_REDIRECT_URI, redirectURI);
-      intent.putExtra(WebViewActivity.EXTRA_RETURN_PROPERTIES, returnProperties);
-      activity.startActivityForResult(intent, WEB_REQUEST_CODE);
+      activity.startActivity(intent);
     }
   }
 
+  public void getAccessToken(IDmeScope scope, IDmeGetAccessTokenListener listener) {
+    AuthToken token = accessTokenManager.getToken(scope);
+    listener.onSuccess(token == null ? null : token.getAccessToken());
+  }
+
+  public void getAccessToken(IDmeScope scope, boolean forceReload, IDmeGetAccessTokenListener listener) {
+    // TODO mirland 13/12/16:
```

Comment:
I will throw an exception, but I will do in another pull. It's other task in jira